fb_apt/attributes/default.rb

```diff
@@ -0,0 +1,22 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+if node.debian?
+  mirror = 'http://http.debian.net/debian'
```

Comment:
that page says specifically you should use httpredir.debian.org/debian - which it turns out is NOT a 404. :)